src/util/tr.js

```diff
@@ -496,75 +499,130 @@ const _tr = (msg, {
       const body = "<div class='wzrkPPdscr' style='color:" + textColor + "'>" + descriptionText + '<div></td></tr></table></div>'
       html = css + title + body
     }
+    if (displayObj['custom-html-sandbox']) {
+      iframe.setAttribute('style', 'z-index: 2147483647; display:block; width: 100% !important; height:100%; border:0px !important; border-color:none !important;')
+    } else {
+      iframe.setAttribute('style', 'z-index: 2147483647; display:block; width: 100% !important; border:0px !important; border-color:none !important;')
+    }
 
-    iframe.setAttribute('style', 'z-index: 2147483647; display:block; width: 100% !important; border:0px !important; border-color:none !important;')
     msgDiv.appendChild(iframe)
-    const ifrm = (iframe.contentWindow) ? iframe.contentWindow : (iframe.contentDocument.document) ? iframe.contentDocument.document : iframe.contentDocument
-    const doc = ifrm.document
 
     // Dispatch event for popup box/banner close
     const closeCampaign = new Event('CT_campaign_rendered')
     document.dispatchEvent(closeCampaign)
 
-    doc.open()
-    doc.write(html)
-
     if (displayObj['custom-editor']) {
-      appendScriptForCustomEvent(targetingMsgJson, doc)
+      html = appendScriptForCustomEvent(targetingMsgJson, html)
+    }
+    iframe.srcdoc = html
+
+    // const ua = navigator.userAgent.toLowerCase()
+    let contentDiv
+
+    const ActionType = {
+      ADJUST_IFRAME_HEIGHT: 'adjustIFrameHeight',
+      UPDATE_HEIGHT: 'update height',
+      GET_NOTIFICATION: 'getnotif',
+      EVENT: 'Event',
+      PROFILE: 'Profile',
+      OUL: 'OUL',
+      CLOSE_BOX_POPUP: 'closeBoxPopUp',
+      CLOSE_BANNER_POPUP: 'closeBannerPopUp'
```

Comment:
same, can you shift this to a different constants file

Reply:
Updated the code